components/suggestion/suggestion_box.jsx

```diff
@@ -517,6 +517,7 @@ export default class SuggestionBox extends React.PureComponent {
                 components: [],
                 selection: '',
             });
+            this.handlePretextChanged("");
```

Comment:
```suggestion
            this.handlePretextChanged('');
```

to fix the linting issue